debug/db.py

```diff
@@ -33,24 +32,11 @@ def __init__(self, settings_file):
         self.db_name = config['DATABASE']['db']
 
         # Establish connection object initialize tables
-        # self._drop_all_tables()
         self._initialize_tables()
-        # self.add_fake_plugin()
-        # self.remove_plugin("https://github.com/BadStreff/slothtorrent_yts")
         # self._add_fake_torrents()
         # self._add_sample_torrents()
         self._connection.close()
 
-    def _drop_all_tables(self):
```

Comment:
I recommend leaving this function and commenting it out, it is useful if we want to re-create the tables in the database.